docs/sample_schema_package/mailman_schema/schema.py

```diff
@@ -13,15 +13,44 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+"""This is an example of a message schema."""
+
 from fedora_messaging import message
 
 
-class Message(message.Message):
+class BaseMessage(message.Message):
+    """
+    You should create a super class that each schema version inherits from.
+    This lets consumers perform ``isinstance(msg, BaseMessage)`` if they are
+    receiving multiple message types and allows the publisher to change the
+    schema as long as they preserve the Python API.
+    """
+
+    def __str__(self):
+        """Return a complete human-readable representation of the message."""
+        return 'Subject: {subj}\n\n{body}\n'.format(
```

Comment:
Not sure about the empty line between the subject and the body, it makes for a very verbose output.

Reply:
Fixed!